test/index.test.js

```diff
@@ -23,44 +20,59 @@ tape('check wrong path throws error', function (t) {
   });
 });
 
-tape('check no error is thrown if we have the user_schema_path key', function (t) {
+tape('Attempt to load a wrong content config schema', function (t) {
   var server = new Hapi.Server();
-  server.connection({ port: 0});
+  server.connection({ port: 0 });
 
   var opts = {
-    user_schema_path: path.join(__dirname, 'schema.json')
+    user_schema_path: path.join(__dirname, 'fixtures', 'wrong_schema.json')
   };
 
-  var schema = require('./schema.json');
-
-  t.deepEqual(schema, {}, 'schema is empty');
-
   server.register({ register: abase, options: opts }, function (err) {
-    t.ok(!err, 'No error thrown');
+    t.ok(err, 'Error thrown');
+    server.stop(function () {
```

Comment:
Cool, I'll look at using beforeEach and afterEach for this https://github.com/substack/tape/issues/59